doc/src/tools/flux.md

```diff
@@ -0,0 +1,239 @@
+# Flux
+
+[Flux](https://flux-rs.github.io/flux) is a refinement type checker
+that can be used to prove user-defined safety properties for Rust.
+Users can write their own properties via refinement type contracts
+(a generalization of pre- and post-conditions). Additionally, out of the
+box, Flux checks for various issues such as
+
+- arithmetic overflows
+- division by zero
```

Comment:
For these, does Flux distinguish between ones causing a panic vs ones deemed undefined behaviour?

Reply:
No, we currently don't distinguish between those.

Reply:
Would you mind including some documentation to this effect?

Reply:
I added a sentence at the end of the section explaining this.